docs/README.md

```diff
@@ -0,0 +1,421 @@
+# Terragrunt website
+
+This is the code for the [Terragrunt website](https://terragrunt.io).
+
+Terragrunt website is built with Jekyll and published on Github Pages from `docs` folder on `master` branch.
+
+# Quick Start
+
+## Download project
+
+Clone or fork Terragrunt [repository](https://github.com/gruntwork-io/terragrunt).
+
+## Run
+
+1. Go to the `docs` folder:
+```
+$ cd docs
+```
+
+2. Install gems:
+```
+$ bundle install
+```
+
+3. Run project:
+```
+$ bundle exec jekyll serve
+```
+
+# Deployment
+
+1. Go to the `docs` folder:
+```
+$ cd docs
+```
+
+2. Build static website
+```
+$ JEKYLL_ENV=production bundle exec jekyll build
+```
+
+3. Commit changes to the repository
+```
+$ git add .
+$ git commit -m "New version"
+$ git push origin <branch>
```

Comment:
Are you suggesting that after a build, you have to commit the generated code? That shouldn't be necessary with GitHub Pages, as it'll run the Jekyll build for you.

Reply:
I forgot to update this section with the last commit. Now it is:

```
# Deployment

GitHub Pages automatically rebuilds the website from the `/docs` folder on the `master` branch.
```